docs/develop/python.mdx

```diff
@@ -80,12 +80,8 @@ from resonate import Resonate
 from resonate.stores.remote import RemoteStore
 from resonate.task_sources.poller import Poller
 
-# highlight-start
-resonate = Resonate(
-    store=RemoteStore(url="http://localhost:8001"), # Resonate Server address
-    message_source=Poller(url="http://localhost:8002"), # Message source configuration
-)
-# highlight-end
+# highlight-next-line
+resonate = Resonate.remote()
```

Comment:
Let's have 1 example that is bare minimum.

And then 1 that shows the customization of the RemoteStore URL and Poller URL?